juju/juju-gui-charm
# Update requirements and Juju GUI to 1.9.2

QA:

```
juju bootstrap
make deploy
```

Verify everything works and there are no JS errors in the console.
